src/app/utils/ledger.ts

```diff
@@ -26,3 +31,10 @@ export const handleBip322LedgerMessageSigning = async ({
 };
 
 export const signatureVrsToRsv = (sig: string): string => sig.slice(2) + sig.slice(0, 2);
+
+export const filterLedgerAccounts = (accounts: Account[], network: NetworkType) =>
+  accounts.filter(
+    (account) =>
+      (account.ordinalsAddress?.startsWith('bc1') && network === 'Mainnet') ||
```

Comment:
Shouldn't this condition be like `account.ordinalsAddress?.startsWith(network === 'Mainnet' ? 'bc1' : 'tb1')` instead?

Reply:
lol, that makes a lot more sense 👍 Thanks 😄